ee/clickhouse/queries/funnels/base.py

```diff
@@ -397,30 +397,24 @@ def _get_cohort_breakdown_join(self) -> str:
 
     def _get_breakdown_conditions(self) -> str:
         if self._filter.breakdown:
-            if self._filter.funnel_step_breakdown:
-                values = self._parse_breakdown_prop_value()
-            else:
-                limit = self._filter.breakdown_limit_or_default
-                first_entity = next(x for x in self._filter.entities if x.order == 0)
-                if not first_entity:
-                    ValidationError("An entity with order 0 was not provided")
-                values = []
-                if self._filter.breakdown_type == "person":
-                    values = get_breakdown_person_prop_values(
-                        self._filter, first_entity, "count(*)", self._team.pk, limit
-                    )
-                elif self._filter.breakdown_type == "event":
-                    values = get_breakdown_event_prop_values(
-                        self._filter, first_entity, "count(*)", self._team.pk, limit
-                    )
+            limit = self._filter.breakdown_limit_or_default
+            first_entity = next(x for x in self._filter.entities if x.order == 0)
```

Comment:
Code issue:

This doesn't work as you'd like:

```
In [2]: next(iter([]))
---------------------------------------------------------------------------
StopIteration                             Traceback (most recent call last)
<ipython-input-2-bfed92c5b1cf> in <module>
----> 1 next(iter([]))

StopIteration: 
```

Given that entities are sorted already entities[0] does the trick